source/features/merge-status.tsx

```diff
@@ -0,0 +1,64 @@
+import './merge-status.css';
+import select from 'select-dom';
+import React from 'dom-chef';
+import * as api from '../libs/api';
+import {getOwnerAndRepo} from '../libs/utils';
+import features from '../libs/features';
+
+const CONFLICTING = 'CONFLICTING';
+
+function buildQuery(
+	ownerName: string,
+	repoName: string,
+	prs: string[],
+): string {
+	return `
+		query {
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				${prs.map(
+		(pr: string) => `
```

Comment:
@fregante This indentation here is a bit off, isn't it?

Reply:
You can write this as
```js
				${prs.map((pr: string) => `
```